src/Features/Core/Portable/Completion/Providers/AbstractDocCommentCompletionProvider.cs

```diff
@@ -223,6 +223,12 @@ protected IEnumerable<CompletionItem> GetTopLevelItems(ISymbol symbol, TSyntax s
                 {
                     items.Add(GetItem(ReturnsElementName));
                 }
+
+                var namedType = symbol as INamedTypeSymbol;
+                if (namedType != null && namedType.IsDelegateType())
+                {
+                    items.AddRange(GetParameterItems(namedType.DelegateInvokeMethod.GetParameters(), syntax, ParameterElementName));
```

Comment:
❗️ Needs to handle `DelegateInvokeMethod` returning `null`:

https://github.com/dotnet/roslyn/blob/9c9e6cae6b8ed91bbe1628f7f8552d7c87249e02/src/Compilers/Core/Portable/Symbols/INamedTypeSymbol.cs#L91-L94